mobile/src/events/ModifyEvent.tsx

```diff
@@ -0,0 +1,138 @@
+import { Formik } from "formik";
+import React, { useState } from "react";
+import { useSelector } from "react-redux";
+import {
+  ScrollView,
+  TextInput,
+  Button,
+  SafeAreaView,
+  StyleSheet,
+} from "react-native";
+import DatePicker from "./DatePicker";
+import * as SecureStore from "expo-secure-store";
+
+const apiUrl = "http://localhost:8000";
+
+const CreateEvent: React.FC<{}> = ({ navigation, route }) => {
```

Comment:
Why is this component called CreateEvent / is it supposed to be replacing/the same as the previous CreateEvent?

Reply:
Nope! Needs to be renamed.